spec.html

```diff
@@ -37,6 +37,151 @@ <h1>Introduction</h1>
+
+  <emu-clause id="sec-nativeerror-object-structure">
+    <h1>_NativeError_ Object Structure</h1>
+    <p>For each error object, references to _NativeError_ in the definition should be replaced with the appropriate error object name from <emu-xref href="#sec-native-error-types-used-in-this-standard"></emu-xref>.</p>
+
+    <emu-clause id="sec-properties-of-nativeerror-instances">
+      <h1>Properties of _NativeError_ Instances</h1>
+      <p>_NativeError_ instances are ordinary objects that inherit properties from their _NativeError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error, _NativeError_, or _AggregateError_ instances.</p>
+    </emu-clause>
+  </emu-clause>
+
+
+  <emu-clause id="sec-aggregate-error-object-structure">
+    <h1>_AggregateError_ Object Structure</h1>
+    <p>For each error object, references to _AggregateError_ in the definition should be replaced with the appropriate error object name from <emu-xref href="#sec-aggregate-error-object-structure"></emu-xref>.</p>
+
+    <emu-clause id="sec-aggregate-error-constructor">
+      <h1>The _AggregateError_ Constructor</h1>
+      <p>The _AggregateError_ constructor:</p>
+      <ul>
+        <li>creates and initializes a new _AggregateError_ object when called as a function rather than as a constructor. A call of the object as a function is equivalent to calling it as a constructor with the same arguments. Thus the function call <code><var>AggregateError</var>(&hellip;)</code> is equivalent to the object creation expression <code>new <var>AggregateError</var>(&hellip;)</code> with the same arguments.</li>
+        <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified _AggregateError_ behaviour must include a `super` call to the _AggregateError_ constructor to create and initialize subclass instances with an [[ErrorData]] internal slot.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error">
+        <h1>AggregateError ( _errors_, _message_ )</h1>
+        <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+          1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+          1. Let _errorsIterator_ be ? GetMethod(_errors_, @@iterator).
+          1. Let _errorsList_ ? IterableToList(_errors_, _errorsIterator_).
+          1. Let _errorsArray_ be CreateArrayFromList(_errorsList_).
+          1. Perform ! CreateDataProperty(_O_, `"errors"`, _errorsArray_).
```

Comment:
```suggestion
          1. Perform ! CreateDataPropertyOrThrow(_O_, `"errors"`, _errorsArray_).
```
Otherwise failure is silently ignored.

Reply:
I was mistakenly copied old algorithm. Just updated semantics to master branch